src/components/template/template.tsx

```diff
@@ -1,61 +1,38 @@
 import { useContext } from "react";
-import Card from "../atoms/card";
 import TaskBar from "../molecules/task-bar";
-import ButtonComponent from "../organism/button-component";
-import { useForm } from "react-hook-form";
-import { yupResolver } from "@hookform/resolvers/yup";
-import { schema } from "../../types/schema";
-import { TTaskState } from "../../types/task-state";
 import { ReducerContext } from "../../lib/initial-reducer";
+import ButtonUI from "../atoms/button";
+import TaskList from "../organism/taskList";
 
 export default function Template() {
-  const { register, handleSubmit, formState, watch } = useForm<TTaskState>({
-    resolver: yupResolver(schema),
-  });
+  const { dispatch } = useContext(ReducerContext);
 
   const { rstate } = useContext(ReducerContext);

```

Comment:
細かいけど、なぜ、8と10行目を分けたん？

```suggestion
  const { rstate, dispatch } = useContext(ReducerContext);
```

Reply:
ごめん、複数行選択しきらんやった。。